cmd/recorder/main.go

```diff
@@ -194,7 +194,9 @@ func doRecord(ctx context.Context, c client.Client, gvks []schema.GroupVersionKi
 
 func forEach(c client.Client, gvk schema.GroupVersionKind, listOptions *client.ListOptions, fn func(unstructured.Unstructured) error) error {
 	for ok := true; ok; ok = listOptions.Continue != "" {
-		if _, ok := opk8s.IgnoredCRDList[gvkToCRDName(gvk)]; ok {
+		crdName := gvkToCRDName(gvk)
+		if _, ok := opk8s.IgnoredCRDList[crdName]; ok {
+			logger.Info(fmt.Sprintf("Orphaned CRD %s found, you should delete it using 'kubectl delete crd %s'", crdName, crdName))
```

Comment:
Logger.Error?

Reply:
I think it is not an error to keep orphaned CRDs. If we use logger.Error() here, then the recorder will act the same as what reported in https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/2874 except for a more actionable message.

Reply:
>  then the recorder will act the same as what reported in #2874 except for a more actionable message.

Right, we expect the users to fix their stale CRD by giving clear guidance and not break their usage. 

Also suggest adding the `logger := klog.FromContext(ctx)`.

Reply:
Done.